plugins/wazo-fanvil/serie-x/pkgs/pkgs.db

```diff
@@ -86,11 +121,21 @@ url: http://download.fanvil.com/Firmware/Release/X3S/20191210/x3s2.10.2.6887T201
 size: 3902427
 sha1sum: 9d16b8936ecf2b1ccd394aaf6e044faef7bb45f1
 
+[file_X3U-fw]
+url: https://download.fanvil.com/Firmware/Release/X3U/20211222/x3u-5912-RECOVERY-P0.18.23.1.23-2.4.12-1164T2021-11-10-01.18.44.z
+size: 11129914
+sha1sum: d18508f3c83d625e4e192dc08bd0d023
```

Comment:
The `sha1sum` are not valid for all the new firmwares
It is not the same as an `md5`